### ISSUE TITLE
[EVM] Use double commit ownership transfer

### ISSUE THREAD

**Issue by nik-suri** [wormhole-foundation/example-native-token-transfers#420]
On Solana, ownership transfer is double commit. This means that the new admin must accept the ownership transfer to finalize it. The old owner can also cancel the ownership transfer before the new admin accepts.

On EVM, ownership transfer is single-shot. This is risky in case the owner accidentally transfers to the wrong address.

We should update EVM ownership transfer to be double commit.

**Comment by gks2022004** [comment]
Yes, we can migrate the Ownable to [Ownable2Step](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable2Step.sol)
